pipeline-templates/job/code-build.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  SolutionBaseName:
+  BuildConfiguration:
+
+jobs:
+- job: CodeBuild
+  variables:
+  - group: BUILD Management Resources
+  steps:
+    - task: gittools.gittools.setup-gitversion-task.gitversion/setup@0

```

Comment:
this name can be simplified now as done [here](https://github.com/NickGraham101/gandt-devops/blob/971ee268e467e883d130405e7612e48ea163a674/AzureDevOpsTemplates/Build/StepTemplates/dotnetcore-build.yml#L21), same for line 15